modules/tinymce/src/core/main/ts/api/UndoManager.ts

```diff
@@ -18,6 +18,10 @@ const UndoManager = (editor: Editor): UndoManager => {
   const locks: Locks = Cell(0);
   const index: Index = Cell(0);
 
+  const extra: Extra = (callback1: () => Promise<void> | void, callback2: () => Promise<void> | void) => {
+    return Rtc.extra(editor, undoManager, locks, index, callback1, callback2) as any;
```

Comment:
Because typescript does not take kindly to a return type of `void | Promise<void>` and starts giving errors that `void` is not `Promise<void>` ( technically correct, but... )